stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -87,6 +89,35 @@ ::gnmi::SubscribeResponse GetResponse(const ::gnmi::Path& path,
   return resp;
 }
 
+// Specialization for '::gnmi::Decimal64'.
+::gnmi::SubscribeResponse GetResponse(const ::gnmi::Path& path,
+                                      const ::gnmi::Decimal64& contents) {
+  ::gnmi::Decimal64* value = TypedDecimalInitializer<::gnmi::Decimal64>(
+      contents.digits(), contents.precision()).InitAllocated();
+
+  ::gnmi::SubscribeResponse resp = GetResponse(path);
+  resp.mutable_update()->mutable_update(0)->mutable_val()
+      ->set_allocated_decimal_val(value);
```

Comment:
Can be replaced by
```c++
::gnmi::SubscribeResponse resp = GetResponse(path);
resp.mutable_update()->mutable_update(0)->mutable_val()->mutable_decimal_val() = contents;
```

Reply:
The `contents` argument is not `T*` but `const T&`.
Could you please explain more detailed about what do you mean here?

Reply:
I forgot a de-reference. Here's the full function:

```c++
// Specialization for '::gnmi::Decimal64'.
::gnmi::SubscribeResponse GetResponse(const ::gnmi::Path& path,
                                      const ::gnmi::Decimal64& contents) {
  ::gnmi::SubscribeResponse resp = GetResponse(path);
  *resp.mutable_update()
      ->mutable_update(0)
      ->mutable_val()
      ->mutable_decimal_val() = contents;
  return resp;
}
```